u8timeseries/timeseries.py

```diff
@@ -141,7 +121,8 @@ def first_value(self) -> float:
         float
             The first value of this series
         """
-        return self._values[0]
+        self._assert_univariate()
```

Comment:
Why do you need to enforce univariate here? All the dimensions in the multivariate case have the same time index, so it would be consistent to just return the first timestamp of this index.